### ISSUE TITLE
Format Steam_ID | Mode of saving player data

### ISSUE THREAD

**Issue by crashzk** [azalty/sm-no-dupe-account#32]
Currently the plugin saves the **`steamID`** data of the players.

![image](https://user-images.githubusercontent.com/32937653/144770119-abdf3977-0c57-4343-b0e0-f06de0801cda.png)

Would it be possible to implement to save the format **`steamID64`** ?

I believe that this makes it easier for the plugin to check the players and even us to locate the players steam.

I ask because in CSGO there are moments that have two saved steamID formats, namely:
- STEAM_0:1:XXXXXXXX
- STEAM_1:0:XXXXXXXX

Anyway, I believe that using the **`steamID64`** format makes the question of checking the player better, or would I be wrong?

**Comment by azalty** [comment]
`STEAM_0` = `STEAM_1`, csgo requires a 1 for some reason, but it's basically the same.

`STEAM_0:1:57298004` = `STEAM_1:1:57298004`. The one starting with 0 is for Steam in general, and starting with a 1 means it's for a special game like csgo. I don't know why it's needed, but csgo requires it.

The next `:0` or `:1` doesn't change from csgo to steam. It is part of the player's steamid, same for the following numbers (`:57298004`)

In short, just change STEAM_1 to STEAM_0 to convert from csgo to Steam.

You can use https://steamid.io or other tools like that to make it easier. This one automatically converts STEAM_1 to STEAM_0.

**Why don't I use steamid64**

I prefer SteamID2 (STEAM_X:X:XXXXXX) because it is shorter and it's basically the default way of identifying someone.
